frontend/packages/employee-frontend/src/components/messages/MessagesPage.tsx

```diff
@@ -83,8 +97,7 @@ export default React.memo(function MessagesPage() {
     setMessageUnderEdit(null)
     setMessageOpen(null)
     if (unit) {
-      loadDraftMessages(unit.id)
-      loadSentMessages(unit.id)
+      loadMessageCounts(unit.id)
```

Comment:
now when you have message list open while creating a draft or sending a message, the list content is not updated

Reply:
Yeah, I guess I'll have to refactor it all again 😆